src/test/tests/databases/zipwrapper.py

```diff
@@ -46,7 +50,7 @@
 DeleteAllPlots()
 CloseDatabase(data_path("zipwrapper_test_data/multi_ucd.visit"))
 
-OpenDatabase(data_path("zipwrapper_test_data/histz_*.silo.gz database"), 3)
+OpenDatabase(data_path("zipwrapper_test_data/histz_*.silo database"), 3)
```

Comment:
So, without the matching extension `.gz`, how does VisIt decide to employ the ZipWrapper plugin? We are not creating a situatiion where ZipWrapper becomes a plugin that winds up *always* being tried during an open, are we?

Reply:
I couldn't figure out why this call would always fail from the cli, yet I could open the virtual database just fine from the gui. Then I noticed that the virtual database as seen in the gui didn't have the .gz.

Reply:
BTW, looking at the log files, the .gz version of the file is used when determining which plugin should be tried, it just doesn't appear in the virtual db listing.